Assignment 3-JS/sass/style.scss

```diff
@@ -0,0 +1,235 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+html {
+  font-size: 80%;
+}
+.container-body {
+  width: 100%;
+  height: 100%;
+  background:linear-gradient(to left,rgb(163, 243, 226),rgb(45, 190, 190),rgb(2, 129, 129));
+  color: white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  .calender {
+      width: 460px;
+      background-color: #00001a;   
+      box-shadow: 5px 5px 10px black;
+      .month {
+          height: 20%;
+          display: flex;
+          justify-content: space-between;
+          align-items: center;
+          text-align: center;
+          padding: 0 20px;
+          border-bottom:2px solid #00e6b8;
+      }
+      .month i {
+          font-size: 20px;
+          cursor: pointer;
+          color: #00e6b8;
+      }
+      .month .date .month-name, .current-year-name {
+          text-transform: uppercase;
+          letter-spacing: 2px;
+          font-size: 30px;
+          color: #00e6b8;
+          cursor: pointer;
+      }
+      .week-days {
+          display: flex;
+          padding: 20px 0px;
+          
+      }
+      .week-days div {
+          width: calc(460px/8);
+          display: flex;
+          justify-content: center;
+          align-items: center;
+      }
+      .days-container {
+          display:flex;
+            #week-number{
+              .week {
+                  width:57.5px;
+                  height:60px;
+                  color:#34b7eb;
+                  display: flex;
+                  justify-content: center;
+                  align-items: center;
+                  box-sizing:border-box;
+              }
+          }
+      }
+      .days {
+          display: flex;
+          flex-wrap: wrap;
+          width:403.3px;
+      }
+      .days div {
```

Comment:
use a class

Reply:
Gave a common class to all div elements inside it's wrapper.